python/cudf/cudf/io/parquet.py

```diff
@@ -203,6 +206,45 @@ def read_parquet(
             )
         filepaths_or_buffers.append(tmp_source)
 
+    if filters is not None:
+        # Convert filters to ds.Expression
+        filters = ioutils.filters_to_expression(filters)
+
+        # Initialize ds.FilesystemDataset
+        dataset = ds.dataset(filepaths_or_buffers, format="parquet")
```

Comment:
You might want to add a `partitioning="hive"` here? (to be compatible with the partitioning that `write_to_dataset` above creates)

(or maybe cudf.read_parquet doesn't support that at all?)